### DISCUSSION TITLE
Commit to revert Pull Request via UI not verified

### DISCUSSION THREAD

**Discussion by dpprdan** [community/community#68048] [Pull Requests]
### Select Topic Area

Bug

### Body

When I [revert a Pull Request via the regular web interface](https://docs.github.com/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/reverting-a-pull-request), the resulting commit is not signed/verified, while any other commit I make via the web interface is verified. 

> GitHub will automatically use GPG to sign commits you make using the web interface. Commits signed by GitHub will have a verified status. 
> https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification

IMHO the revert commits should be signed as well.

**Comment by dpprdan** [comment]
It seems that this depends on whether the original PR (the one to be reverted) was merged with a simple merge commit or a squash and merge. 

Compare 

https://github.com/dpprdan/revert_verified_test/pull/2, which reverts https://github.com/dpprdan/revert_verified_test/pull/1, which was merged with a simple merge commit

with

https://github.com/dpprdan/revert_verified_test/pull/4, which reverts https://github.com/dpprdan/revert_verified_test/pull/3, which was merged with a squash and merge.

https://github.com/dpprdan/revert_verified_test/pull/4/commits/d6cab27d8771e3bf383378682968b8fd5f8e35a5 is not signed/verified, but should IMO.

**Comment by othmane399** [comment]
same bug for us, really annoying as we cannot enforce signature for all members...

**Comment by somal-vora** [comment]
Is this possible to solve ? revert should be single quicks and with signed ideally.

**Reply by othmane399** [reply]
For us it was fixed since github have announced rotation of some of their internal keys, so you should maybe retry ? on our side we enforced signed commits on all organization couples of weeks ago and there's no issue now.
